src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,40 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../types/Post';
```

Comment:
It's a good practice to keep types in a separate file. Consider moving `Post` type to a new file called 'types.ts' in the 'types' folder 📁 and then import it here.